src/test/debuginfo/pretty-huge-vec.rs

```diff
@@ -10,10 +10,10 @@
 // gdb-command: run
 
 // gdb-command: print vec
-// gdb-check:$1 = Vec<u8>(len: 1000000000, cap: 1000000000) = {[...]...}
+// gdb-check:$1 = len=1000000000 = {[...]...}
```

Comment:
hmm. Is this change to the print output intended? It seems unfortunate to stop emitting the type information, but maybe there is some motivation that I am not seeing here?

Reply:
@pnkfelix Thank you for notice, I've missed it because LLDB emits type name automatically before `$1`. So, I think it's better to emit only container type names and just ignore generic type names. In this case, emit `Vec` instead of `Vec<u8>`. It's better IMO because generic types may be really huge and useless, e.g. in the case of `HashMap`:
```Rust
HashMap<i32, i32, std::collections::hash_map::RandomState>
```
and I don't know a way to render it more pretty. Also, default C++ STL pretty-printers ignore template type names and only produces `std::vector` for `std::vector<int>`.

Thus, I'd prefer using reduced type names hardcoded right inside `to_string` method of each container pretty-printer, like this:
```Python
class StdVecProvider:
    def __init__(self, valobj):
        ...
    def to_string(self):
        return "Vec(size={})".format(self.length)
```

What do you think?

Reply:
Yeah, including the container type name but leaving out the type parameters sounds okay to me.